base/scripts/code.game.cs

```diff
@@ -423,10 +423,24 @@ function onServerGhostAlwaysDone()
 
             Client::buildMenu(%clientId, "Pick a team:", "InitialPickTeam");
                 Client::addMenuItem(%clientId, "0Observe", -2);
-                Client::addMenuItem(%clientId, "1Automatic", -1);
-                for(%i = 0; %i < getNumTeams(); %i = %i + 1)
-                    Client::addMenuItem(%clientId, (%i+2) @ getTeamName(%i), %i);
-                    %clientId.justConnected = "";
+                
+                //remove automatic option in tourney mode
+                if(!$Server::TourneyMode) {
+                    
+                    Client::addMenuItem(%clientId, "1Automatic", -1);
+                    

```

Comment:
nit: remove extra blank lines
```suggestion
                    Client::addMenuItem(%clientId, "1Automatic", -1);
```